src/main/java/org/gradle/profiler/GradleBuildConfiguration.java

```diff
@@ -60,10 +60,9 @@ public void runGradle(String... arguments) {
     }
 
     public void addGradleCommand(List<String> commandLine) {
-        if (OperatingSystem.isWindows()) {
-            commandLine.add("cmd.exe");
-            commandLine.add("/C");
-        }
-        commandLine.add(new File(gradleHome, "bin/gradle").getAbsolutePath());
+        String gradleScriptName = OperatingSystem.isWindows()
+            ? "gradle.bat"
+            : "gradle";
+        commandLine.add(new File(gradleHome, "bin/" + gradleScriptName).getAbsolutePath());
```

Comment:
How about adding a test where the project directory contains a space?

Reply:
That wouldn't cause a problem. We only have problems here when either the Gradle installation or the Gradle home is in a spaced location.

We don't have a test for a locally installed Gradle version, but I made the test specifying a Gradle home to use one with a space in the name.

Reply:
Understood.